src/clarity_cli.rs

```diff
@@ -471,27 +469,6 @@ pub fn vm_execute(program: &str, clarity_version: ClarityVersion) -> Result<Opti
     })
 }
 
-fn save_coverage(
```

Comment:
I'm surprised code coverage reporting is being dropped entirely.  Would it make more sense to re-implement the code coverage code we have now as an implementation of `EvalHooks`?  Then we won't be dropping any features.